### PR TITLE
ENH get loaded marker geometries through Atracsys wrapper

### PR DESCRIPTION
**Pull request by atracsys-sbt**
The geometry that describes the tracked markers can be retrieved as a vector of coords x,y,z (float) for each fiducial that composes the marker.

### CODE REVIEW

File: src/PlusDataCollection/Atracsys/AtracsysTracker.cxx [hunk 1]
```diff
@@ -127,6 +127,8 @@ class AtracsysTracker::AtracsysInternal
   // helper function to load ftkGeometry from string
   ATRACSYS_RESULT LoadFtkGeometryFromString(const std::string& geomString, ftkGeometry& geom);
 
+  std::map<int, std::vector<std::array<float, 3>>> geometries;
```
Comment (Sunderlandkyl) [inline src/PlusDataCollection/Atracsys/AtracsysTracker.cxx:130]: geometries should be capitalized.

File: src/PlusDataCollection/Atracsys/vtkPlusAtracsysTracker.h [hunk 2]
```diff
@@ -33,15 +33,17 @@ class vtkPlusDataCollectionExport vtkPlusAtracsysTracker : public vtkPlusDevice
...
+  * leftIntrinsic = left camera focal length [0-1], optical center [2-3], lens distorsion [4-8] and skew [9]
+  * rightIntrinsic = left camera focal length [0-1], optical center [2-3], lens distorsion [4-8] and skew [9]
+  * rightPosition = position of the right camera in the coordinate system of the left camera
+  * rightOrientation = orientation of the right camera in the coordinate system of the left camera
+  */
   PlusStatus GetCamerasCalibration(
     std::array<float, 10>& leftIntrinsic, std::array<float, 10>& rightIntrinsic,
     std::array<float, 3>& rightPosition, std::array<float, 3>& rightOrientation);
 
+  PlusStatus GetLoadedGeometries(std::map<int, std::vector<std::array<float, 3>>>& geometries);
```
Comment (Sunderlandkyl) [inline src/PlusDataCollection/Atracsys/vtkPlusAtracsysTracker.h:45]: Add a comment describing the function, and the data returned in "geometries".

File: src/PlusDataCollection/Atracsys/AtracsysTracker.cxx (hunk 1 above)
Comment (atracsys-sbt) [reply src/PlusDataCollection/Atracsys/AtracsysTracker.cxx]: done

File: src/PlusDataCollection/Atracsys/vtkPlusAtracsysTracker.h (hunk 2 above)
Comment (atracsys-sbt) [reply src/PlusDataCollection/Atracsys/vtkPlusAtracsysTracker.h:49]: done

**Pull request merged**